deps/openssl/openssl_common.gypi

```diff
@@ -1,4 +1,8 @@
 {
+  'variables': {
```

Comment:
Not necessary since this is an included file (`*.gypi`) and we have 
https://github.com/nodejs/node/blob/b4e670dc26b50271457a257ccb244b14ff323b64/common.gypi#L28